lib/turn.rb

```diff
@@ -0,0 +1,24 @@
+# turn.rb
+require_relative 'card'
+class Turn
+  attr_accessor :guess, :card, :correct
+
+  def initialize(guess, card)
+    @guess = guess
+    @card = card
+  end
+
+  def correct?
+    @guess.downcase.capitalize == card.answer.downcase.capitalize ? true : false
+  end
+
+  def feedback?
+    if correct? == true
```

Comment:
Because a call to `correct?` returns a boolean you do not need to compare to a boolean to make sure it is `true`

These both are equivalent:

```ruby
if correct? == true
```

```ruby
if correct?
```